### DISCUSSION TITLE
In Calculating liquidity, what's the left and right segment of start point?

### DISCUSSION THREAD

**Discussion by sowells** [Jeiwan/uniswapv3-book#83] [Milestone 1. First Swap]
Hello! 
I'm studying UniswapV3, but I'm having some difficulties. 

> Now, break the curve from the image above into two segments: one to the left of the start point and one to the right of the start point. 
...
the left segment is made entirely of token x, the right segment is made entirely of token y. 

I couldn't imagine what the left segment and right segment were. Is this correct? (Maybe it isn't.)

![image](https://github.com/Jeiwan/uniswapv3-book/assets/25666319/179d188e-5423-44a1-9c75-b00c7b0903f0)

**Comment by t0x1cC0de** [comment]
Here you go-

![image](https://github.com/Jeiwan/uniswapv3-book/assets/20126340/5900fced-7789-4928-9bd8-155ed8bb163a)
